common/buildings/wc_duchy_capital_buildings.txt

```diff
@@ -0,0 +1,315 @@
+
+# REGULAR DUCHY BUILDINGS
+
+aerodrome_01 = {
+	construction_time = slow_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		building_requirement_civilized = yes
+		OR = {
+			scope:holder.culture = {
+				OR = {
+					has_innovation = innovation_taming_the_skies
+					has_innovation = innovation_flying_beasts
+			}
+			culture_has_light_aerial_maa = yes
+			culture_has_massive_aerial_maa = yes
+		}
+	}
+	
+	can_construct_showing_failures_only = {
+		
+		culture = {
+			has_innovation = innovation_burhs
+		}
+	}
+	
+	is_enabled = {
+		county.duchy.holder = county.holder
+	}
+	show_disabled = yes
+	
+	cost_gold = expensive_building_tier_2_cost
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.3
+		light_aerial_toughness_mult = 0.3
+		light_aerial_screen_mult = 0.3
+		light_aerial_pursuit_mult = 0.3
+		
+		massive_aerial_damage_mult = 0.3
+		massive_aerial_toughness_mult = 0.3
+		massive_aerial_screen_mult = 0.3
+		massive_aerial_pursuit_mult = 0.3
+		
+		men_at_arms_maintenance = -0.2
+	}
+	
+	next_building = aerodrome_02
+	
+	type_icon = "icon_building_aerodrome.dds"
+
+	type = duchy_capital
+	
+	ai_value = {
+		base = 20
+		modifier = {
+			factor = 2
+			scope:holder.capital_province = this
+		}
+	}
+}
+
+aerodrome_02 = {
+	construction_time = slow_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		building_requirement_civilized = yes
+	}
+	
+	can_construct_showing_failures_only = {
+		
+		culture = {
+			has_innovation = innovation_castle_baileys
+		}
+	}
+	
+	is_enabled = {
+		county.duchy.holder = county.holder
+	}
+	show_disabled = yes
+	
+	cost_gold = expensive_building_tier_3_cost
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.4
+		light_aerial_toughness_mult = 0.4
+		light_aerial_screen_mult = 0.4
+		light_aerial_pursuit_mult = 0.4
+		
+		massive_aerial_damage_mult = 0.4
+		massive_aerial_toughness_mult = 0.4
+		massive_aerial_screen_mult = 0.4
+		massive_aerial_pursuit_mult = 0.4
+		
+		knight_limit = 1
+		
+		men_at_arms_maintenance = -0.5
+	}
+	
+	next_building = aerodrome_03
+	
+	type = duchy_capital
+	
+	ai_value = {
+		base = 20
+		modifier = {
+			factor = 2
+			scope:holder.capital_province = this
+		}
+	}
+}
+
+aerodrome_03 = {
+	construction_time = slow_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		building_requirement_civilized = yes
+	}
+	
+	can_construct_showing_failures_only = {
+		
+		culture = {
+			has_innovation = innovation_royal_armory
+		}
+	}
+	
+	is_enabled = {
+		county.duchy.holder = county.holder
+	}
+	show_disabled = yes
+	
+	cost_gold = expensive_building_tier_4_cost
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.5
+		light_aerial_toughness_mult = 0.5
+		light_aerial_screen_mult = 0.5
+		light_aerial_pursuit_mult = 0.5
```

Comment:
I think we need to make it more expensive too.
I can suggest increasing mult value costs by 2.
Just add the multiplication to `heavy_infantry` mults. Costs of the rest units are linked to them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/159191181-1d454598-b0d7-41b0-a415-2b173788e3a2.png)

</details>

Reply:
Isn't increasing their overall power by 50% at least a tad too much? That sounds overpowerdly broken to me...

Reply:
@zumbak04 ^^?

Reply:
> I think we need to make it more expensive too. I can suggest increasing mult value costs by 2. Just add the multiplication to `heavy_infantry` mults. Costs of the rest units are linked to them.
> 
> Click to expand
> ![image](https://user-images.githubusercontent.com/46384992/159191181-1d454598-b0d7-41b0-a415-2b173788e3a2.png)

Do you mean multiply each _mult by 2?

Reply:
> Do you mean multiply each _mult by 2?

Yes.